### DISCUSSION TITLE
Why is the dbt power user extension pane only working in the .sql files, not .yml files?

### DISCUSSION THREAD

**Discussion by janmechtel** [AltimateAI/vscode-dbt-power-user#604] [Q&A] [answered]
I frequently wish I had the information of the extension when working on the .yml file.

The first idea I had was to move everything from the .yml into the .sql, but it seems that dbt doesn't support that. For example the model properties like columns: and desc: can not be defined inside a config macro in the .sql file.

What alternative is there?

How can I quickly get from a .yml file to the corressponding model.sql?

**Comment by mdesmet** [comment, marked as answer]
What you describe seems useful. Can you create a feature request describing in detail what you are looking for?

**Comment by janmechtel** [comment]
https://github.com/AltimateAI/vscode-dbt-power-user/issues/607 hope this is what you had in mind
